qa/suites/fs/upgrade/featureful_client/upgraded_client/tasks/4-compat_client.yaml

```diff
@@ -9,7 +9,17 @@ tasks:
       - ceph fs required_client_features cephfs add metric_collect
 - sleep:
     duration: 5
+# client.0 is upgraded and client.1 is evicted by the MDS due to missing
+# feature compat set
 - fs.clients_evicted:
     clients:
       client.0: False
       client.1: True
+# cleanup evicted client so there's no trouble later
+- exec:
+    client.1:
+      - echo $(mount | grep ceph-fuse | awk '{print $3}') > mntpt.txt
+      - cat mntpt.txt
+      - sudo umount -f $(cat mntpt.txt)
+      - sudo rmdir $(cat mntpt.txt)
+      - rm mntpt.txt
```

Comment:
How about do this in `tasks/fs.py` instead ? Then it should be very simple by:

`mount.umount_wait()`

Reply:
Could it lead to a problem for teuthology jobs that wants to keep evicted client as it is around for some time?

Reply:
There are only two place using this, I didn't see it could lead potential issues.

Reply:
And from my understanding the `fs.clients_evicted:` is where needs the evicted clients as it is around for some time. So after this or in this to unmount them should be fine.

Reply:
Okay, I'll make the change in that case.

Reply:
You'd only need to make sure that the `findmnt` (from your changes) does not interleave anywhere, which then could lead to a a test hang.

Reply:
Done. PTAL.